helmfile/charts/cluster-admin-rbac/values.yaml

```diff
@@ -0,0 +1 @@
+users: []
```

Comment:
Just out of curiosity, what happens if you deploy with this empty? Is that still valid?

Reply:
Tried it out and yes, it is still valid. It will give you a `ClusterRoleBinding` with no subjects.